### PR TITLE
Updated servicePlanInfo with list of possible status values.

### PR DESCRIPTION
**Pull request by piotrci**
As requested by @viv-liu

### CODE REVIEW

File: api-reference/beta/resources/serviceplaninfo.md [hunk 1]
```diff
@@ -8,8 +8,8 @@ Contains information about a service plan associated with a subscribed SKU. The
 |:---------------|:--------|:----------|
 |servicePlanId|Guid|The unique identifier of the service plan.|
 |servicePlanName|String|The name of the service plan.|
-|provisioningStatus|String|The provisioning status of the service plan.|
-|appliesTo|String||
+|provisioningStatus|String|The provisioning status of the service plan. Possible values:<br/>"Success" - Service is fully provisioned.<br/>"Disabled" - Service has been disabled.<br/>"PendingInput" - Service is not yet provisioned; awaiting service confirmation.<br/>"PendingActivation" - Service is provisioned but requires explicit activation by administrator (e.g. Intune_O365 service plan)<br/>"PendingProvisioning" - Microsoft has added a new service to the product SKU and it has not been activated in the tenant, yet.|
```
Comment (Lauragra) [inline api-reference/beta/resources/serviceplaninfo.md]: Please add a period after the "PendingActivation" entry, and change "e.g." to "for example".

Comment (piotrci) [inline api-reference/beta/resources/serviceplaninfo.md]: Done.

**Comment by viv-liu** [comment]
Are we ready to merge?

File: api-reference/v1.0/resources/serviceplaninfo.md [hunk 2]
```diff
@@ -8,8 +8,8 @@ Contains information about a service plan associated with a subscribed SKU. The
 |:---------------|:--------|:----------|
 |servicePlanId|Guid|The unique identifier of the service plan.|
 |servicePlanName|String|The name of the service plan.|
-|provisioningStatus|String|The provisioning status of the service plan.|
-|appliesTo|String||
+|provisioningStatus|String|The provisioning status of the service plan. Possible values:<br/>"Success" - Service is fully provisioned.<br/>"Disabled" - Service has been disabled.<br/>"PendingInput" - Service is not yet provisioned; awaiting service confirmation.<br/>"PendingActivation" - Service is provisioned but requires explicit activation by administrator (e.g. Intune_O365 service plan)<br/>"PendingProvisioning" - Microsoft has added a new service to the product SKU and it has not been activated in the tenant, yet.|
```
Comment (Lauragra) [inline api-reference/v1.0/resources/serviceplaninfo.md]: Change the "(e.g. Intune_O365 service plan)" text to "(for example, Intune_O365 service plan)".

Comment (piotrci) [inline api-reference/v1.0/resources/serviceplaninfo.md]: Done.

**Pull request merged**